integrations/integration-vip-config.php

```diff
@@ -80,11 +80,23 @@ private function set_config( string $slug ): void {
 	protected function get_vip_config_from_file( string $slug ) {
 		$config_file_path = ABSPATH . 'config/integrations-config/' . $slug . '-config.php';
 
-		if ( ! is_readable( $config_file_path ) ) {
+		/**
+		 * Clear realpath cache to get new path else we can get cached path which can be non-existant
+		 * because k8s configmaps updates the file via symlink instead of actually replacing the file and
+		 * PHP cache can hold a reference to the old symlink.
+		 *
+		 * Did tried using `clearstatcache( true, $config_file_path )` but it was giving old symlink,
+		 * `is_readable()` does return correct value (false) for this case but we prefer
+		 * to get the config which `clearstatcache( true )` does return because it gives us new symlink.
+		 */
+		clearstatcache( true );
```

Comment:
Clearing the whole file status cache doesn't seems like a good idea but none of the other solution were working like I have also mentioned in comment so please let me know if there is something else I should try.

Reply:
Added changes to clear cache of a specific file.